### ISSUE TITLE
Abnormal circulating Cartilage Oligomeric Matrix Protein concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5991]
**Preferred term label:** Abnormal circulating Cartilage Oligomeric Matrix Protein concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cartilage oligomeric matrix protein in the blood is outside the normal range., PMID: 30100245

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Cartilage Oligomeric Matrix Protein concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cartilage oligomeric matrix protein in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Cartilage Oligomeric Matrix Protein concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Cartilage Oligomeric Matrix Protein concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cartilage oligomeric matrix protein in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Cartilage Oligomeric Matrix Protein concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
As far as I can see, there is no defined normal range, so that for an individual patient, it is not possible to say that a COMP level is abnormal. Published studies show increases in cohorts. If there is a defined normal range, please reopen this term and provide e.g. a LOINC term and a PMID, thanks!
